resources/definitions/renkforce_rf100.def.json

```diff
@@ -0,0 +1,231 @@
+            "default_value": 1.0
+        },
+        "raft_base_thickness": {
+            "default_value": 0.3
+        },
+        "raft_interface_line_spacing": {
+            "default_value": 3.0
+        },
+        "raft_interface_line_width": {
+            "default_value": 0.4
+        },
+        "raft_interface_thickness": {
+            "default_value": 0.27
+        },
+        "raft_margin": {
+            "default_value": 5.0
+        },
+        "raft_surface_layers": {
+            "default_value": 2.0
+        },
+        "raft_surface_line_spacing": {
+            "default_value": 3.0
+        },
+        "raft_surface_line_width": {
+            "default_value": 0.4
+        },
+        "raft_surface_thickness": {
+            "default_value": 0.27
+        },
+        "retraction_amount": {
+            "default_value": 2.0
+        },
+        "retraction_combing": {
+            "default_value": "all"
+        },
+        "retraction_enable": {
+            "default_value": true
+        },
+        "retraction_hop_enabled": {
+            "default_value": 1.0
+        },
+        "retraction_min_travel": {
+            "default_value": 1.5
+        },
+        "retraction_speed": {
+            "default_value": 40.0
+        },
+        "skin_overlap": {
+            "default_value": 15.0
+        },
+        "skirt_brim_minimal_length": {
+            "default_value": 150.0
+        },
+        "skirt_gap": {
+            "default_value": 3.0
+        },
+        "skirt_line_count": {
+            "default_value": 1.0
+        },
+        "speed_infill": {
+            "default_value": 50.0
+        },
+        "speed_layer_0": {
+            "default_value": 30.0
+        },
+        "speed_print": {
+            "default_value": 50.0
+        },
+        "speed_topbottom": {
+            "default_value": 30.0
+        },
+        "speed_travel": {
+            "default_value": 50.0
+        },
+        "speed_wall_0": {
+            "default_value": 25.0
+        },
+        "speed_wall_x": {
+            "default_value": 35.0
+        },
+        "support_angle": {
+            "default_value": 60.0
+        },
+        "support_enable": {
+            "default_value": 0.0
+        },
+        "support_infill_rate": {
+            "default_value": 15.0
+        },
+        "support_pattern": {
+            "default_value": "lines"
+        },
+        "support_type": {
+            "default_value": "everywhere"
+        },
+        "support_xy_distance": {
+            "default_value": 0.5
+        },
+        "support_z_distance": {
+            "default_value": 0.1
+        },
+        "top_thickness": {
+            "default_value": 0.5
+        },
+        "wall_thickness": {
+            "default_value": 0.8
+        }
+    },
+
+    "categories": {
+        "material": {
+            "settings": {
+                "material_bed_temperature": {
+                    "visible": false
```

Comment:
You should just put this `"material_bed_temperature"` in the overrides along with the rest. It doesn't matter that this one edits the `visible` property rather than `default_value`. Perhaps you would put a white space between them.